pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -106,7 +104,15 @@ stdenv.mkDerivation {
       else ./mips64-default-n64.patch)
   ;
 
-  outputs = [ "out" "info" "man" ];
+  outputs = [ "out" "info" "man" "dev" ]
+  # Ideally we would like to always install 'lib' into a separate
+  # target. Unfortunately cross-compiled binutils installs libraries
+  # across both `$lib/lib/` and `$out/$target/lib` with a reference
+  # from $out to $lib. Probably a binutils bug: all libraries should go
```

Comment:
I agree.

I fought with this for a few hours trying to fix up all the contorted paths that binutils' `make install` creates and finally gave up.